### PR TITLE
Upgrade to Scala 2.11.5

### PR DESCRIPTION
**Pull request by cb372**
@gidsg

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -4,6 +4,10 @@ name := "sanity-tests"
 
 version := "1.0"
 
+scalaVersion := "2.11.5"
```
Comment (mchv) [inline build.sbt]: `2.11.6` is now out :)

File: app/com/gu/contentapi/sanity/utils/QuartzScheduler.scala [hunk 2]
```diff
@@ -9,14 +9,10 @@ import org.quartz.impl.StdSchedulerFactory
...
-import scala.concurrent.duration.Duration
 import java.lang.Class
 
-import javax.tools.ToolProvider
-
-import  scala.collection.mutable.Map
-import  scala.collection.mutable.HashMap
-import  scala.collection.mutable.SynchronizedMap
+import  scala.collection.concurrent.TrieMap
+import scala.concurrent.duration.Duration
```
Comment (gidsg) [inline app/com/gu/contentapi/sanity/utils/QuartzScheduler.scala]: spacing is not consistent

File: build.sbt [hunk 3]
```diff
@@ -4,6 +4,10 @@ name := "sanity-tests"
...
+
+scalacOptions ++= Seq("-feature")
```
Comment (gidsg) [inline build.sbt]: Do we need this or was it just for debugging purposes?

File: build.sbt (hunk 1 above)
Comment (cb372) [inline build.sbt]: Wow! 26th Feb. I totally missed that.

File: app/com/gu/contentapi/sanity/utils/QuartzScheduler.scala (hunk 2 above)
Comment (cb372) [inline app/com/gu/contentapi/sanity/utils/QuartzScheduler.scala]: fixed

File: build.sbt (hunk 3 above)
Comment (cb372) [inline build.sbt]: It was to debug a feature warning. But I think it's useful to leave it in there. I always enable this flag for Scala projects. 

Without it, you get compiler warnings saying "be careful, you used a smarty pants feature of Scala" but without any useful info about why the warning was generated.

Comment (gidsg) [inline build.sbt]: OK, makes sense.

**Comment by gidsg** [comment]
:+1: :rainbow:

**Pull request merged**